docs/Other SDL Features/Calling a Phone Number/index.md

```diff
@@ -56,17 +56,20 @@ func isDialNumberSupported(handler: @escaping (_ success: Bool, _ error: Error?)
         return handler(true, nil)
     }
 
-    // Check if the module supports the `DialNumber` request
+    // Check if the phone capability has already been retrieved from the module
+    if let phoneCapability = sdlManager.systemCapabilityManager.phoneCapability {
+        // Check if the module supports the `DialNumber` request
```

Comment:
I don't believe that this is the correct comment here.

Reply:
Could you clarify what the correct comment would be?